src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -185,21 +186,58 @@ public class FhirR4 {
       Config.getAsBoolean("exporter.fhir.use_shr_extensions");
   protected static boolean TRANSACTION_BUNDLE =
       Config.getAsBoolean("exporter.fhir.transaction_bundle");
+
+  /* For the most part the US Core changes from v4 -> v5 are backwards compatible,
+   * so for those cases just check USE_US_CORE_IG.
+   * If there are version differences then use USE_US_CORE_(#) to separate them out.
+   */
   protected static boolean USE_US_CORE_IG =
       Config.getAsBoolean("exporter.fhir.use_us_core_ig");
+  protected static String US_CORE_VERSION =
+      Config.get("exporter.fhir.us_core_version", "");
```

Comment:
Seems like you'd want to have a default value, otherwise you won't be using any version of US Core, right?

Reply:
Added 5.0.1 as the default value, so it's the same as the default value in the properties file